### DISCUSSION TITLE
How to create rolling backups - daily, monthly, yearly prune

### DISCUSSION THREAD

**Discussion by yonas** [rustic-rs/rustic#1186] [Q&A] [answered]
Borg has a `prune` command that allows you to keep up to X daily snapshots, Y monthly snapshots, and Z yearly snapshots. Borg will remove the oldest snapshots.

Can rustic do this? If not, I'll submit a feature request.

**Comment by intgr** [comment, marked as answer]
Yes, the `rustic forget` command follows your configured snapshot retention policy.

See Rustic [Forget Options configuration documentation](https://github.com/rustic-rs/rustic/tree/main/config#forget-options).

**Reply by aawsome** [reply]
@yonas just to clarify: with rustic, the forget command removes snapshots, the prune command removes the unused data, typical after a `forget` run. 

Note, that there is also `rustic forget --prune` which does both in one run.
